extensions/wikia/Recirculation/RecirculationController.class.php

```diff
@@ -76,7 +71,7 @@ public function footer() {
 		global $wgSitename, $wgCityId, $wgLanguageCode;
 
 		// Language code check is temporary to supress MCF for old discussions on non-en wikis;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)